packages/cli/src/databases/migrations/common/1730386903556-CreateTestEntityTable.ts

```diff
@@ -0,0 +1,35 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+const testEntityTableName = 'test_entity';
+
+export class CreateTestEntityTable1730386903556 implements ReversibleMigration {
+	async up({ schemaBuilder: { createTable, column } }: MigrationContext) {
+		await createTable(testEntityTableName)
+			.withColumns(
+				column('id').int.notNull.primary.autoGenerate,
+				column('name').varchar(255).notNull,
+				column('workflowId').varchar(36).notNull,
+				column('evaluationWorkflowId').varchar(36),
+				column('annotationTagId').varchar(16),
+			)
+			.withIndexOn('workflowId')
+			.withIndexOn('evaluationWorkflowId')
+			.withForeignKey('workflowId', {
+				tableName: 'workflow_entity',
+				columnName: 'id',
+				onDelete: 'CASCADE',
+			})
+			.withForeignKey('evaluationWorkflowId', {
+				tableName: 'workflow_entity',
+				columnName: 'id',
+			})
+			.withForeignKey('annotationTagId', {
+				tableName: 'annotation_tag_entity',
+				columnName: 'id',
+			}).withTimestamps;
```

Comment:
Is it intentional that these don't have `onDelete`?

Reply:
These two references are nullable, and we don't want to delete a test when an evaluation workflow or specific annotation tag being deleted. So the right setting should be `ON DELETE SET NULL` I guess?

Reply:
I mean it's completely fine to omit it, just wondering if that was done intentionally or forgotten